BlueWizard/SpeechSynthesizer.mm

```diff
@@ -68,36 +75,66 @@ -(Buffer *)bufferFor:(NSArray *)wrappedSamples {
 }
 
 -(void)fillBuffer:(NSMutableArray *)samples {
-    unsigned int frames = _tms5220->m_fifo_count;
-    short int buffer[frames];
-    _tms5220->process(buffer, frames);
+    short int buffer[SAMPLES_PER_HALF_FRAME];
+    _tms5220->process(buffer, SAMPLES_PER_HALF_FRAME);
     float scale = 1.0f / (1 << 15);
-    for (int i = 0; i < frames; i++) {
+    for (int i = 0; i < SAMPLES_PER_HALF_FRAME; i++) {
         [samples addObject:[NSNumber numberWithDouble:buffer[i] * scale]];
     }
 }
 
 -(void)writeData:(int)data {
-    _tms5220->wsq_w(1);
+    //_tms5220->wsq_w(1); // don't use these since we want to use the 5220 without worrying about timing the reads and writes
     _tms5220->data_w(data);
-    _tms5220->wsq_w(0);
+    //_tms5220->wsq_w(0);
```

Comment:
definitely would prefer to not have old unused code littered in the codebase.  I don't think these comments are really useful, if its needed to see what was there before, there's git history.